server/models/report.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+module.exports = (sequelize, DataTypes) => {
+  const Report = sequelize.define('Report', {
+    id: {
+      type: DataTypes.UUID,
+      primaryKey: true,
+      defaultValue: DataTypes.UUIDV4,
+      unique: true,
+      allowNull: false,
+    },
+    comment: {
+      type: DataTypes.TEXT,
+      allowNull: false,
+    },
+    open: {
+      type: DataTypes.BOOLEAN,
+      allowNull: false,
+      defaultValue: true,
+    },
+  }, {});
+  Report.associate = function (models) {
+    // associations can be defined here
+    Report.belongsTo(models.Article, {
+      foreignkey: 'articleId',
+      as: 'article',
+      onDelete: 'CASCADE',
+    });
+    Report.belongsTo(models.User, {
+      foreignkey: 'userId',
+      as: 'user',
+      onDelete: 'CASCADE',
+    });
+    Report.belongsTo(models.ReportType, {
+      foreignkey: 'typeId',
```

Comment:
I think we should stick to 'foreignKey'

Reply:
@Chiazokam Thanks for catching this. I will fix this.